src/planscape/config/conditions_config.py

```diff
@@ -17,8 +17,58 @@
 class PillarConfig:
     """
     Class wrapping the configuration of regions, pillars, elements, and metrics.
+    This is represented as a multi-level dictionary.  The top level is the region
+    dictionary, mapping region names to data about the region.  Each region contains
+    a list of pillars, each of which is also a dictionary, etc.
     """
 
+    # Keys in the dictionaries.
+    COMMON_METADATA = {'filepath', 'display_name',
+                       'colormap', 'max_value', 'min_value'}
+    REGION_KEYS = {'region_name', 'pillars'}.union(COMMON_METADATA)
+    PILLAR_KEYS = {'pillar_name',
+                   'elements',
+                   'operation',
+                   'display'}.union(COMMON_METADATA)
+    ELEMENT_KEYS = {'element_name',
+                    'metrics',
+                    'operation'}.union(COMMON_METADATA)
+    METRIC_KEYS = {'metric_name',
+                   'current_conditions_only',
+                   'ignore',
+                   'source',
+                   'source_link',
+                   'data_source',
```

Comment:
if this is always a set of people or an organization, would data_owner be more descriptive?

Reply:
I agree---"data_source" is too vague.  How about "data_provider"?

Reply:
SG